ldes-server-application/pom.xml

```diff
@@ -45,7 +45,17 @@
             <dependencies>
                 <dependency>
                     <groupId>be.vlaanderen.informatievlaanderen.vsds</groupId>
-                    <artifactId>ldes-server-infra-mongo</artifactId>
+                    <artifactId>mongo-repository</artifactId>
+                    <version>${project.version}</version>
+                </dependency>
+            </dependencies>
+        </profile>
+        <profile>
+            <id>changeset-mongo</id>
+            <dependencies>
+                <dependency>
+                    <groupId>be.vlaanderen.informatievlaanderen.vsds</groupId>
+                    <artifactId>mongock-changeset-1</artifactId>
```

Comment:
Are we sure that we want a separate module for every changeset instead of one 'changeset module' with all changesets in it? With separate modules:
- Over time the changesets will clutter the profile list.
- The Dockerfile will need to be updated every time as well and will grow as well.

Reply:
but isn't that why we have a config file specifying the changesets to be run?